### PR TITLE
50% of the fix for scroll volume in mini-player

### LINKED ISSUE #494: Mini player volume control should be configurable
**OS:**
10.11.3
**Issue Descriptions:**
Mini player settings: should have option to disable "mouse scroll changes player volume"
**Steps to Reproduce:**
* start a track
* launch mini player
* scroll mouse

### PR DESCRIPTION
**Pull request by jostrander**
Fixes #494, but requires GPMDP/gmusic-mini-player.js/pull/7

### CODE REVIEW

**Comment by berryelectronics** [comment]
Just a Quick question: will it show the English string if it is missing in
the other language options?

Jesse Ostrander <notifications@github.com> schrieb am Mo., 30. Mai 2016,
01:25:

> Fixes #494
> <https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/494>,
> but requires GPMDP/gmusic-mini-player.js#7
> <https://github.com/GPMDP/gmusic-mini-player.js/pull/7>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>
> https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/pull/1125
> Commit Summary
>
>    - 50% of the fix for scroll volume in mini-player
>
> File Changes
>
>    - *M* src/_locales/en-US.json
>    <https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/pull/1125/files#diff-0>
>    (1)
>    - *M* src/inject/GPMWebView/interface/mini.js
>    <https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/pull/1125/files#diff-1>
>    (7)
>    - *M* src/inject/settings/mini.js
>    <https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/pull/1125/files#diff-2>
>    (13)
>    - *M* src/main/features/core/desktopSettings.js
>    <https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/pull/1125/files#diff-3>
>    (3)
>    - *M* src/public_html/desktop_settings.html
>    <https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/pull/1125/files#diff-4>
>    (4)
>
> Patch Links:
>
>    -
>    https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/pull/1125.patch
>    -
>    https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/pull/1125.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/pull/1125>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AMCBvPesQSfK6OgVjsBau7q3E48ufr7Sks5qGiB6gaJpZM4IpZ4S>
> .
>
-- 
Matjas

**Comment by jostrander** [comment]
It should, yes, but it will fail CI entirely unless every translation is done.

File: src/inject/GPMWebView/interface/mini.js [hunk 1]
```diff
@@ -69,3 +72,7 @@ Emitter.on('miniAlwaysOnTop', (event, state) => {
     remote.getCurrentWindow().setAlwaysOnTop(state.state);
   }
 });
+Emitter.on('miniUseScrollVolume', (event, state) => {
+  console.log(state.state); // eslint-disable-line
```
Comment (MarshallOfSound) [inline src/inject/GPMWebView/interface/mini.js]: Somebody left a `console` statement in :crying_cat_face:

Comment (jostrander) [inline src/inject/GPMWebView/interface/mini.js]: Whoops, you got me. :gun:

**Comment by MarshallOfSound** [comment]
@jostrander Also

* The CI will fail while missing translations :+1: 
* But it won't replace it with the english version, it will replace it with `Unknown Translation`

**Comment by jostrander** [comment]
My bad, I thought I saw the code say it failed back somewhere. I knew the tests would fail due to missing translations however.

**Pull request merged**